doc/source/users/macro_report.rst

```diff
@@ -0,0 +1,14 @@
+.. highlight:: python
+   :linenothreshold: 5
+   
+.. currentmodule:: sardana.macroserver.macros.logging
+
+
+.. _sardana-macros-reports:
+
+=============
+Macro Reports
+=============
+
+:ref:`Report messages <sardana-macro-reporting>` can be send by the macros to a report file. This report feature is configured as described :ref:`here <sardana-configuration-macroserver>`. 
```

Comment:
We should [avoid hyperlinks like *here*](http://www.writethedocs.org/guide/writing/docs-principles/#skimmable)